examples/test_tetra3.py

```diff
@@ -30,13 +29,14 @@ def draw_box(img_draw, centre, radius, **kwargs):
 t3 = tetra3.Tetra3(load_database='default_database')
 # t3 = tetra3.Tetra3(load_database='database_60_90')
 
-# Select method used for star detection and centroiding. True for cedar-detect,
-# False for Tetra3.
-USE_CEDAR_DETECT = True
+# Use cedar_detect if we are able to load it.
+try:
+    from tetra3 import cedar_detect_client
+    cedar_detect = cedar_detect_client.CedarDetectClient()
+    USE_CEDAR_DETECT = True
+except:
+    USE_CEDAR_DETECT = False
```

Comment:
this automatically tries to use cedar-detect if the client is able to initialize, otherwise uses default tetra3 code.